### PR TITLE
Project capstone

### PR DESCRIPTION
**Pull request by cilfonegabriel**
The purpose of this project is the presentation of the conference that will be given by Barcelo FC.
We worked with two sections, one called HOME and the other ABOUT, both with two different css, js and html.
The README file was also created, containing all the information about the project, including an explanatory video.
First, we proceeded to work on the mobile version and then the desktop version. For the Desktop version, media queries was used, and also the cards of the presenters of the desktop version were created with js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,96 @@
...
+Give a ⭐ if you like this project!
+
+## Acknowledgments
+
+- Hard work
+- Inspiration
+- etc
+
+## Explication about the project and look at the website
+Here video: [Gabriel Cilfone](https://www.loom.com/share/894ab4daec184cd685fc27d8d9594ad0)
```
Comment (Ismailco) [inline README.md]: - [x] The video should contain a highlight about a piece of the code you find interesting or is proud of making, kindly consider showing and talking about some piece of code you proud of

File: index.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                <img src="./assets/images/logo_barc-removebg-preview.png" alt="">
+                <ul class="nav-list">
+                <li><a class="about" href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="OS-campaign" href="#">OS Campaign</a></li>
+            </ul>
+        </nav>
```
Comment (Ismailco) [inline index.html:42]: - [x] The navigation bar is taking up too much space kindly consider reducing the size of the navigation bar so it looks more like the provided design

Screenshot_20221201_214924

File: index.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    </section>
+    <footer>
+        <div class="mobile-about-footer">
+            <img src="./assets/images/logo_barc-removebg-preview.png" alt="logo">
+            <div class="footer-p">
+            <p>2021 Open Source Global summit. All right reserved</p><br>
+           <p> Grow with open source Africa!</p>
+        </div>
+        </div>
+    </footer>
```
Comment (Ismailco) [inline index.html:146]: - [x] The Footer is taking up too much space kindly consider reducing the size of it so it looks more like the provided design

|Your page|Provided design|
|-|-|
|Screenshot_20221201_215259|Screenshot_20221201_215519|

File: about.html [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        </section>
+        <footer>
+            <div class="mobile-about-footer">
+                <img src="./assets/images/logo_barc-removebg-preview.png" alt="logo">
+                <div class="footer-p">
+                <p>2023 Week of the Barcelona Innovation Center. All rights reserved.</p><br>
+               <p> Take your jump to a new level!</p>
+            </div>
+            </div>
+        </footer>
```
Comment (Ismailco) [inline about.html:112]: - [x] The footer on the about page should have a darker background, kindly consider implementing this so your page looks exactly as the provided design

|Your page|Provided design|
|-|-|
|Screenshot_20221201_215259|Screenshot_20221201_215618|

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @cilfonegabriel, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: assets/styles/style.css [hunk 5]
```diff
@@ -0,0 +1,722 @@
...
+  .mobile-about-footer {
+    background-color: var(--secondary_2);
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 0 0 0 0;
+    width: 100%;
+    height: 76px;
+  }
+
```
Comment (levy002) [inline assets/styles/style.css:671]: - [ ] As requested by the previous reviewer, the footer on the about page should have a darker background, kindly consider implementing this so your page looks exactly as the provided design. Please make sure to use the same color as the one on provided Behance design.

   - ### YOUR DESIGN
      image

   - ### PROVIDED DESIGN
   
      image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @cilfonegabriel   , :wave:
### I'm Levy, your PR reviewer

Excellent work :sunglasses: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @cilfonegabriel, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights

- Linter checks are passing
- Good commit messages
- Requested changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**